setup.py

```diff
@@ -7,6 +7,7 @@
 import shutil
 import sys
 
+from pip.req import parse_requirements
```

Comment:
How will this affect pre-installation dependencies?

![image](https://cloud.githubusercontent.com/assets/618416/16785768/ad9ace60-4844-11e6-8a4d-878fca929386.png)
http://stackoverflow.com/questions/14399534/how-can-i-reference-requirements-txt-for-the-install-requires-kwarg-in-setuptool#comment26169011_16624700